lib/src/text_input_dialog/ios_text_input_dialog.dart

```diff
@@ -31,6 +33,7 @@ class IOSTextInputDialog extends StatefulWidget {
   final bool useRootNavigator;
   final WillPopCallback? onWillPop;
   final bool autoSubmit;
+  final Widget Function(Widget)? textInputBuilder;
```

Comment:
`textFields` is List<DialogTextField>` and `textInputBuilder` isn't List, so I think it's not a good API.

Reply:
Yes, but how I would change the text decoration, that's why I added the builder method

Reply:
It certainly works, but I don't want to increase the complexity of this package by adding that customizability with its API design.